plugins/makerModeFilter.go

```diff
@@ -132,21 +133,16 @@ func (f *makerModeFilter) filterOps(
 	numKeep := 0
 	numDropped := 0
 	numTransformed := 0
-	filteredOps := []build.TransactionMutator{}
+	filteredOps := []txnbuild.Operation{}
 	for _, op := range ops {
-		var newOp build.TransactionMutator
+		var newOp txnbuild.Operation
 		var keep bool
 		switch o := op.(type) {
-		case *build.ManageOfferBuilder:
+		case *txnbuild.ManageSellOffer:
 			newOp, keep, e = f.transformOfferMakerMode(baseAsset, quoteAsset, topBidPrice, topAskPrice, o)
 			if e != nil {
 				return nil, fmt.Errorf("could not transform offer (pointer case): %s", e)
 			}
-		case build.ManageOfferBuilder:
```

Comment:
I think we need this too - sometimes the value was not a pointer. is that not possible with `txnbuild.Operation`?

Reply:
This will always be a pointer because the structs that implement the `txnbuild.Operation` interface are pointers